web/src/utils/constants.js

```diff
@@ -0,0 +1,16 @@
+const dandiUrl = 'https://dandiarchive.org';
+const dandiAboutUrl = 'https://dandiarchive.org/about';
+const dandiDocumentationUrl = 'https://www.dandiarchive.org/handbook/10_using_dandi/';
+
+const schemaRelease = process.env.VUE_APP_DANDISET_SCHEMA_RELEASE || '1.0.0-rc1';
```

Comment:
You make a good point, but I think it's fine to leave for now, since once https://github.com/dandi/dandi-api/pull/83 is merged, this logic will change to fetch the schema url from the API anyway.